whois-api/src/main/java/net/ripe/db/whois/api/autocomplete/AutocompleteSearch.java

```diff
@@ -88,17 +95,22 @@ public AutocompleteSearch(final FullTextIndex fullTextIndex) {
                 for (ScoreDoc scoreDoc : topDocs.scoreDocs) {
                     final Document doc = indexSearcher.doc(scoreDoc.doc);
                     final Map<String, Object> result = Maps.newLinkedHashMap();
-                    result.put("key", doc.get(FullTextIndex.LOOKUP_KEY_FIELD_NAME));
-                    result.put("type", doc.get(FullTextIndex.OBJECT_TYPE_FIELD_NAME));
+
+                    final RpslObject rpslObject =   objectDao.getByKey(
```

Comment:
Handle EmptyResultDataAccessException here (MariaDB and Lucene may be out of sync)